tests/platform_tests/api/test_fan_drawer_fans.py

```diff
@@ -320,13 +303,14 @@ def test_set_fans_speed(self, duthosts, enum_rand_one_per_hwsku_hostname, localh
                     target_speed = random.randint(speed_minimum, speed_maximum)
 
                 speed = fan_drawer_fan.get_speed(platform_api_conn, j, i)
-                speed_tol = fan_drawer_fan.get_speed_tolerance(platform_api_conn, j, i)
 
                 speed_set = fan_drawer_fan.set_speed(platform_api_conn, j, i, target_speed)     # noqa F841
-                time.sleep(5)
+                time.sleep(self.get_fan_facts(duthost, j, i, 5, "speed", "delay"))
 
                 act_speed = fan_drawer_fan.get_speed(platform_api_conn, j, i)
-                self.expect(abs(act_speed - target_speed) <= speed_tol,
+                under_speed = fan_drawer_fan.is_under_speed(platform_api_conn, j, i)
```

Comment:
Can we also check over_speed > under_speed?

Reply:
`is_under_speed` and `is_over_speed` return boolean values; a comparison wouldn't make sense here.

Reply:
Got it. thanks